ledger/src/blockstore.rs

```diff
@@ -3601,8 +3600,11 @@ impl Blockstore {
     /// Returns a mapping from each elements of `slots` to a list of the
     /// element's children slots.
     pub fn get_slots_since(&self, slots: &[Slot]) -> Result<HashMap<Slot, Vec<Slot>>> {
-        let slot_metas: Result<Vec<Option<SlotMeta>>> =
-            self.meta_cf.multi_get(slots.to_vec()).into_iter().collect();
+        let slot_metas: Result<Vec<Option<SlotMeta>>> = self
+            .meta_cf
+            .multi_get(slots.iter().copied())
```

Comment:
This one is unfortunate - I'd like to update `get_slot_since()` to accept a `Vec<Slot>` instead of `&[Slot]`. Can pull that into this PR or do separately ... I initially left it out as it would modify public function in `Blockstore` / will require an update to the calling location in `ReplayStage`

Reply:
Yeah, unfortunate. Updating the `get_slot_since` signature in a separate PR seems good, though.